src/main/java/org/jboss/forge/arquillian/command/TestSetUp.java

```diff
@@ -0,0 +1,15 @@
+package org.jboss.forge.arquillian.command;
+
+
+import org.jboss.forge.roaster.model.source.JavaClassSource;
+import org.jboss.forge.roaster.model.source.MethodSource;
+
+public interface TestSetUp {
+
+    default void removeTestMethod(String methodName, JavaClassSource test) {
```

Comment:
@bartoszmajsak I think we need to remove method which has annotation @Deployment. I believe we have to remove the same method for all types i.e. docker, k8s & algeron as well. If we move this logic to a component offering this functionality of test removal then we have to repeat it again which is violating DRY.